### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by yizhouyu** [aliciayuting/ORIE4741-project#12]
This project is about automating detection of default clients for banks or other financial institutions. The data set that they are using comes from a Kaggle competition. The features of their data include credit card holders’ registration information and their transaction data. The goal of this project is to decide whether a client would default or not.

What I like about the proposal:
- The team tried various algorithms, from simple ones such as perceptron to more complex ones such as XGBoost and neural networks. They also have done extensive work in feature engineering in order to tackle specific challenges (PCA in order to reduce feature space, and feature normalization in order to improve the neural network)
- The team clearly defined the three error metrics that they used, and for each model that they mentioned, they recorded those three metrics in order to show improvement as they used more sophisticated models.
- The team successfully addressed the challenge that the dataset is imbalanced, with only 20% of the total samples as default customers. They up-balanced the imbalanced dataset and yielded good results.
- The report has a good discussion on whether the model is a weapon of math destruction.

Areas for improvement:
- The team can add a few lines in the abstract about their findings / the performance of their best model, in order to show up front how well they did.
- The report can include a bit more discussions as to what features are most important to the prediction task. If the more complex models are less interpretable, the team can report their findings on simpler models.
- The group can add some reasoning as to why some models perform better than others in the context of default prediction.
